app/services/authentication/providers.rb

```diff
@@ -1,10 +1,3 @@
-# We require all authentication modules to make sure providers
```

Comment:
moved to an an initializer

Reply:
It's great that we could remove this from here 🚀